examples/GetDefaultRetention.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2019 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import io.minio.GetDefaultRetentionArgs;
+import io.minio.MinioClient;
+import io.minio.errors.MinioException;
+import io.minio.messages.ObjectLockConfiguration;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+public class GetDefaultRetention {
+  /** MinioClient.getDefaultRetention() example. */
+  public static void main(String[] args)
+      throws IOException, NoSuchAlgorithmException, InvalidKeyException {
+    try {
+      /* Amazon S3: */
+      MinioClient s3Client =
+          new MinioClient("https://s3.amazonaws.com", "YOUR-ACCESSKEYID", "YOUR-SECRETACCESSKEY");
+
+      ObjectLockConfiguration config =
```

Comment:
IMO we should have a complete example, like creating the bucket, setting the default retention and then fetching it. So that  user just have to edit the credentials and run the file.

Reply:
I think we need to be specific to examples for an API. Its not possible to run any example without modifying essentials like endpoint, credentials, bucket/object names, respective variables etc and some operations are chargeable in some service providers.

We wouldn't consider examples as tests.